src/parser.rs

```diff
@@ -60,36 +60,60 @@ pub fn parse_term(sig: &mut Signature, input: &str) -> Result<Term, ParseError>
 ///
 /// # TRS syntax
 ///
-/// `input` is parsed as a `<program>`, defined as follows:
+/// The input is parsed as a `program`, defined as follows in [augmented Backus-Naur form]:
```

Comment:
Why move to augmented BNF?

Reply:
My main motivation was the readability of incremental alternatives (the lines with `left /= alternative`).